search-commons/src/main/java/no/unit/nva/search/constants/ApplicationConstants.java

```diff
@@ -17,32 +21,35 @@ public final class ApplicationConstants {
 
     public static final String TICKETS_INDEX = "tickets";
 
-    public static final List<AggregationDto> RESOURCES_AGGREGATIONS = List.of(
-        new AggregationDto("entityDescription.reference.publicationInstance.type",
+    public static final List<AbstractAggregationBuilder<? extends AbstractAggregationBuilder<?>>>
+            RESOURCES_AGGREGATIONS = List.of(
+
+        generateSimpleAggregation("entityDescription.reference.publicationInstance.type",
                            "entityDescription.reference.publicationInstance.type.keyword"),
-        new AggregationDto("resourceOwner.owner",
+        generateSimpleAggregation("resourceOwner.owner",
                            "resourceOwner.owner.keyword"),
-        new AggregationDto("resourceOwner.ownerAffiliation",
+        generateSimpleAggregation("resourceOwner.ownerAffiliation",
                            "resourceOwner.ownerAffiliation.keyword"),
-        new AggregationDto("entityDescription.contributors.identity.name",
+        generateSimpleAggregation("entityDescription.contributors.identity.name",
                            "entityDescription.contributors.identity.name.keyword"),
-        new AggregationDto("topLevelOrganization.id",
-                           "topLevelOrganization.id.keyword")
-    );
-
-    public static final List<AggregationDto> TICKETS_AGGREGATIONS = List.of(
-        new AggregationDto("type",
-                           "type.keyword"),
-        new AggregationDto("status",
-                           "status.keyword")
+        generateObjectLabelsAggregation("topLevelOrganization")
     );
-
-    public static final List<String> TICKET_INDICES =
-        List.of(DOIREQUESTS_INDEX, MESSAGES_INDEX, PUBLISHING_REQUESTS_INDEX);
+    public static final String LABELS = "labels";
+    public static final String KEYWORD = "keyword";
+    public static final String ENGLISH = "en";
+    public static final String BOKMAAL = "nb";
+    public static final String NYNORSK = "nn";
```

Comment:
The variable name could indicate that its a language code, and not something else. LANGUAGE_CODE_ENGLISH, ect. They could also be stored together as a Enum, or in a map.